config-connector/solutions/projects/helm/project-services/README.md

```diff
@@ -0,0 +1,97 @@
+# Project Services
+
+==================================================
+
+## NAME
+
+  project-services
+
+## SYNOPSIS
+
+  Config Connector compatible YAML files to enable services on a desired project.
+
+## CONSUMPTION
+
+  1. Clone GoogleCloudPlatform/cloud-foundation-toolkit repository:
+
+      ```bash
+      git clone https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git
+      ```
+
+  1. Go to the project-services folder:
+
+      ```bash
+      cd cloud-foundation-toolkit/config-connector/solutions/projects/helm/project-services
+      ```
+
+## REQUIREMENTS
+
+1. GKE Cluster with Config Connector and [Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#enable_workload_identity_on_a_new_cluster).
+
+1. Cloud Resource Manager API enabled in the project where Config Connector is installed.
+
+1. A working Config Connector cluster using the "cnrm-system" service account must have `roles/serviceusage.serviceUsageAdmin` or `roles/owner` for the desired project.
+
+1. [Helm](../../../README.md#helm)
+
+## USAGE
+
+All steps are run from the current directory ([config-connector/solutions/projects/helm/project-services](.)).
+
+1. Review and update the values in `./values.yaml`.
+
+1. Create a namespace. If you want to use your existing namespace skip this step and use your own namespace name instead of `project-annotated` in all other steps.
+
+    ```bash
+    kubectl create namespace project-annotated
+    ```
+
+1. Validate and install the sample with Helm.`PROJECT_ID` should be a new project ID unique within GCP.
```

Comment:
Project already exists, so this should read something like "should be the project ID of the desired project" instead of "should be a new project..."